modules/apps/store/extensions/assets/webapp/pages/asset.jag

```diff
@@ -120,11 +120,26 @@ require('/modules/store.js').exec(function (ctx) {
 
         var defaultApplicationName = "DefaultApplication";
 
-        var enterpriseSubscription = subscriptionService.getSubscription(appInfo, defaultApplicationName, "ENTERPRISE", authenticator.getLoggedInUser().username).subscription; 
+        var enterpriseSubscription;
```

Comment:
This code has been changed in the latest code. (logic/where condition has been removed)